src/main/resources/db/migration/V8__submission_add_column_rpa_status.sql

```diff
@@ -0,0 +1,2 @@
+CREATE TYPE rpa_status_type AS ENUM('submission_ready', 'in_progress', 'failed', 'success');
+ALTER TABLE submissions ADD COLUMN rpa_status rpa_status_type;
```

Comment:
Do you think this should be a new table? Bethany was saying in an office hours that ideally we shouldnt alter the submissions table

Reply:
Yes - it could be in a separate table which is linked by id. However this sort of alteration might be ok because our app will not directly modify the value of this column (that would be on Jorge's end)